ticket_group_details.view.lkml

```diff
@@ -0,0 +1,60 @@
+view: ticket_group_details {
```

Comment:
Isn't this `ticket_group_facts` or what is `_details`?

Reply:
This is group details about the touches.  Updating to `ticket_touches_groups`

Reply:
`ticket_group_touches`